stablehlo/tests/interpret_convolution.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+func.func @convolution_op_test_si64() {
+  %lhs = stablehlo.constant dense<[[
+                                    [[1], [2], [5], [6]],
+                                    [[3], [4], [7], [8]],
+                                    [[10], [11], [14], [15]],
+                                    [[12], [13], [16], [17]]
+                                  ]]> : tensor<1x4x4x1xi64>
+  %rhs = stablehlo.constant dense<1> : tensor<3x3x1x1xi64>
+  %result = stablehlo.convolution(%lhs, %rhs)
+    dim_numbers = [b, 0, 1, f]x[0, 1, i, o]->[b, 0, 1, f],
+    window = {
+      stride = [4, 4],
+      lhs_dilate = [2, 2]
+    } {
+      batch_group_count = 1 : i64,
+      feature_group_count = 1 : i64,
+      precision_config = [#stablehlo<precision DEFAULT>, #stablehlo<precision DEFAULT>]
```

Comment:
Can this be elided? Since you're using the defaults. If not, we should consider using [DefaultValuedAttr](http://google3/third_party/llvm/llvm-project/mlir/include/mlir/IR/CommonAttrConstraints.td;l=93;rcl=602520726) (in a future change)

Reply:
The interpreter currently doesn't handle different variations of `precision_config`, so this makes it explicit for now. There's also a couple of issues (#445, #755, #879) to figure out the exact story for `precision_config`.